src/View/Window.vala

```diff
@@ -449,6 +472,30 @@ namespace Marlin.View {
             content.add_view (mode, location);
         }
 
+        private int location_is_duplicate (GLib.File location, out bool is_child) {
+            is_child = false;
+            string parent_path = "";
+            string uri = location.get_uri ();
+            /* Ensures consistent format of protocol and path */
+            parent_path = PF.FileUtils.get_parent_path_from_path (location.get_path ());
+            int existing_position = -1;
```

Comment:
Wouldn't it be better to initialize this to 0 and then increment `existing_position` at the end of the `foreach`?

Reply:
Done